### DISCUSSION TITLE
欢迎来到「OSCAR·开源之书·共读」讨论

### DISCUSSION THREAD

**Discussion by lijiangsheng1** [oscar-open-source-book/website#1] [Announcements]
欢迎来到「OSCAR·开源之书·共读」讨论！

**Comment by songshi1225** [comment]
1.范老师的PPT中，有一页是讲“开源文化的未来发展，可能不是一个局部的问题 ，应该从社会结构中去思考”。首先是文化，其次是在开源的这个群体中，是怎么发展及落地的，是一个圈层的共性问题。只是在中国本土中的发展和思考。
2.建胜老师，讲到的时间线的事件，只是罗列，并不能解释事件发展的内在逻辑，同时也不是线性发展。
3.下次讨论是不是可以聚焦些，尝试解决一些实际问题。比如，以互联网为主的公司，公司战略层是怎么看待这个问题的；中国自己的开源工具，包括开放量子基金这种组织，怎么全球化发展；

**Reply by lijiangsheng1** [reply]
话题的把控，全靠参与者自己影响了～ 没有人可以定义～
